zaius/s3/utils.py

```diff
@@ -0,0 +1,63 @@
+import boto3
+import os
+from functools import partial
+from multiprocessing import Pool, cpu_count
+
+import zaius.auth as auth
+
+class S3Client():
```

Comment:
maybe we can get rid of this class? 
from a user perspective when using this module it might look like:
```python
from zaius import s3

s3.upload_file(auth,...)
s3.download_file(auth,...)
s3.list_files(auth,...)
```

Reply:
Yer sorry I should have said I am already working to do that